src/main/java/teammates/common/datatransfer/FeedbackSessionResultsBundle.java

```diff
@@ -295,7 +289,7 @@ public FeedbackSessionResultsBundle(FeedbackSessionAttributes feedbackSession,
         this.emailNameTable = emailNameTable;
         this.emailLastNameTable = emailLastNameTable;
         this.emailTeamNameTable = emailTeamNameTable;
-        this.instructorEmailNameTable = getInstructorEmailNameTableFromRoster(roster);
+        this.commentGiverEmailToNameTable = roster.getEmailToNameTableFromRoster();
```

Comment:
Changing this would mean lot of other related changes in already implemented comments feature.
For eg one of the attributes of `FeedbackResponseCommentRow` is this table.  Didn't we want to keep that side untouched while implementing for feedback participants? Also I have removed emailname/ teamname tables from roster and implemented as you suggested.

Reply:
Understood. But this suggestion applied your new comment feature :)